patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -395,7 +464,21 @@
  
              if (!this.field_70170_p.field_72995_K)
              {
-@@ -2700,7 +2777,10 @@
+@@ -2656,6 +2751,13 @@
+                 this.field_184628_bn = 0;
+             }
+         }
++        if (WATCHER_POTION_EFFECTS.equals(p_184206_1_) && field_70170_p.field_72995_K) {
++            NBTTagList nbttaglist = this.field_70180_af.func_187225_a(WATCHER_POTION_EFFECTS).func_150295_c("ActiveEffects", 10);
++    		field_70713_bf.clear();
++    		for (int i = 0; i < nbttaglist.func_74745_c(); ++i) {
++    			NBTTagCompound nbttagcompound = nbttaglist.func_150305_b(i);
```

Comment:
...

Reply:
Now I am really stumped about this one. This is definitely needed. Should I have put it in one line?